python/tvm/contrib/hexagon/pytest_plugin.py

```diff
@@ -84,10 +84,13 @@ def android_serial_number() -> Optional[str]:
 def get_free_port() -> int:
     """Return the next port that is available to listen on"""
     global PREVIOUS_PORT
+    random.seed(0)
```

Comment:
this should only run once otherwise it resets the rng state and you get the same value every time (not sure if that's okay here or not), maybe guard this behind a global flag `RNG_SEEDED` or something

```
>>> import random
>>> random.seed(0)
>>> random.random()
0.8444218515250481
>>> random.seed(0)
>>> random.random()
0.8444218515250481
```